application/src/main/java/org/togetherjava/tjbot/features/help/HelpSystemHelper.java

```diff
@@ -202,11 +203,17 @@ private RestAction<Message> useChatGptFallbackMessage(ThreadChannel threadChanne
     }
 
     void writeHelpThreadToDatabase(long authorId, ThreadChannel threadChannel) {
+
+        Instant createdAt = Instant.now();
+        String tag = threadChannel.getAppliedTags().getFirst().getName();
```

Comment:
are you suggesting in config or just in code?

Reply:
in the code, like ur only writing the first tag in the db, y not save  all

Reply:
The main reason i only went with one tag is because our helper classes choose the most appropriate tag for a  thread, so if multiple tags are added only helper from one category are invited to thread. Although it makes sense to store data with all tags if this will be used for showcasing stats.

I'm not too sure about implementation details, ill look into it how to handle this.

Reply:
> I'm not too sure about implementation details, ill look into it how to handle this.

using csv/bsv will be easiest without changing much